docs/content/concepts/01-handlers.md

```diff
@@ -0,0 +1,240 @@
++++
+title = "Request Handlers and Routing"
+slug = "handlers"
+weight = 1
++++
+
+## Introduction
+
+Request handlers are a fundamental building block of Boltzmann applications.
+They are functions which you provide to Boltzmann to **handle** incoming HTTP
+requests.
+
+<!-- more -->
+
+This document will cover how to write handler functions, how to control request
+routing, how handler responses are interpreted, and how to control these
+behaviors using attributes on your exported functions.
+
+## What is a Request Handler?
+
+A request handler is any function exported from `handlers.js` (or
+`handlers/index.js`) that accepts a Boltzmann [`Context`] object. A request
+handler is called when an incoming HTTP request matches against its `.route`
+attribute.
```

Comment:
It's true `<alec guinness>`from a certain point of view.`</alec guinness>`. These are handlers that will never get dispatched because there's no matching route. I'm gonna take another stab at this sentence, though – I want to mention that it takes `Context` to set up the next graf.